src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/BulkPrintStaffNotificationService.java

```diff
@@ -0,0 +1,81 @@
+package uk.gov.hmcts.cmc.claimstore.services.staff;
+
+import com.google.common.collect.ImmutableList;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.cmc.claimstore.config.properties.emails.StaffEmailProperties;
+import uk.gov.hmcts.cmc.claimstore.documents.content.bulkprint.BulkPrintEmailContentProvider;
+import uk.gov.hmcts.cmc.claimstore.services.staff.models.EmailContent;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.email.EmailAttachment;
+import uk.gov.hmcts.cmc.email.EmailData;
+import uk.gov.hmcts.cmc.email.EmailService;
+import uk.gov.hmcts.reform.pdf.service.client.PDFServiceClient;
+import uk.gov.hmcts.reform.sendletter.api.Document;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static java.util.Objects.requireNonNull;
+import static uk.gov.hmcts.cmc.claimstore.utils.DocumentNameUtils.buildDefendantLetterFileBaseName;
+import static uk.gov.hmcts.cmc.claimstore.utils.DocumentNameUtils.buildSealedClaimFileBaseName;
+import static uk.gov.hmcts.cmc.email.EmailAttachment.pdf;
+
+@Service
+public class BulkPrintStaffNotificationService {
+
+    private final EmailService emailService;
+    private final StaffEmailProperties emailProperties;
+    private final BulkPrintEmailContentProvider emailContentProvider;
+    private final PDFServiceClient pdfServiceClient;
+
+    @Autowired
+    public BulkPrintStaffNotificationService(
+        EmailService emailService,
+        StaffEmailProperties emailProperties,
+        BulkPrintEmailContentProvider emailContentProvider,
+        PDFServiceClient pdfServiceClient
+    ) {
+        this.emailService = emailService;
+        this.emailProperties = emailProperties;
+        this.emailContentProvider = emailContentProvider;
+        this.pdfServiceClient = pdfServiceClient;
+    }
+
+    public void notifyFailedBulkPrint(List<Document> documents, Claim claim) {
+        EmailContent emailContent = emailContentProvider.createContent(wrapInMap(claim));
+
+        EmailAttachment defendantLetter = pdf(
+            createPdf(documents.get(0)),
+            buildDefendantLetterFileBaseName(claim.getReferenceNumber())
+        );
+
+        EmailAttachment sealedClaim = pdf(
+            createPdf(documents.get(1)),
```

Comment:
Can you please check if we have an automatic test which verifies that the documents will always appear in this order?

Reply:
there is an unit test

Reply:
The test is useless in that matter, refactoring can easily bring breaking change that is not detected. See my comment above.